website/www/site/content/en/documentation/sdks/python-machine-learning.md

```diff
@@ -192,7 +192,9 @@ with pipeline as p:
 
 For more information on resource hints, see [Resource hints](/documentation/runtime/resource-hints/).
 
-## Use a keyed ModelHandler
+## RunInference Patterns
```

Comment:
We should include text after this header describing the section. Something along the lines of:

This section provides examples of RunInference patterns.

(But hopefully more informative than that.)

Reply:
Done - let me know if you think it sounds ok: "This section suggests patterns and best practices that can be used to make your inference pipelines simpler, more robust, and more efficient."